src/ipc/ipc4/handler.c

```diff
@@ -1063,6 +1063,16 @@ void ipc_boot_complete_msg(struct ipc_cmd_hdr *header, uint32_t data)
 	header->ext = 0;
 }
 
+void ipc_send_panic_notification(void)
+{
+	msg_notify.header = SOF_IPC4_NOTIF_HEADER(SOF_IPC4_EXCEPTION_CAUGHT);
```

Comment:
`msg_notify` only exists if `CONFIG_LOG_BACKEND_ADSP_MTRACE` is selected

Reply:
Can you share more information ? I don't get it . Thanks!